bootstrap/_gentoo_common.sh

```diff
@@ -0,0 +1,12 @@
+PACKAGES="dev-vcs/git dev-lang/python:2.7 dev-python/virtualenv dev-util/dialog app-admin/augeas dev-libs/openssl dev-libs/libffi app-misc/ca-certificates virtual/pkgconfig"
+case "$PACKAGE_MANAGER" in 
```

Comment:
This is a system var.

Gentoo has a tool called `eselect` it let's user set preferred package manager. This var is used by other tools that need to interact with package manager. This is exactly that case.